nntrainer/compiler/recurrent_realizer.cpp

```diff
@@ -128,7 +129,8 @@ static void propagateTimestep(LayerNode *node, unsigned int time_step,
   /** @todo add an interface to check if a layer supports a property */
   auto is_recurrent_type = [](LayerNode *node) {
     return node->getType() == LSTMLayer::type ||
-           node->getType() == LSTMCellLayer::type;
+           node->getType() == LSTMCellLayer::type ||
+           node->getType() == RNNCellLayer::type;
```

Comment:
It isn't just for the cell implementation. if it is, let's add others. GRU as well.

Reply:
This is only for the cell implementation. LSTMLayer supports unoptimized LSTMcell implementation, but RNNLayer and GRULayer do not. GRUCell will be added here once its ready.